addons/hr_holidays/models/hr_employee_base.py

```diff
@@ -330,7 +330,7 @@ def _compute_allocation_remaining_display(self):
                         employee_remaining_leaves += virtual_remaining_leaves\
                             if leave_type.request_unit in ['day', 'half_day']\
                             else virtual_remaining_leaves / (employee.resource_calendar_id.hours_per_day or HOURS_PER_DAY)
-                        employee_max_leaves += allocation.number_of_days
+                        employee_max_leaves += allocation.number_of_days if allocation.holiday_status_id.active else 0
```

Comment:
```suggestion
                        if leave_type.requires_allocation == 'no':
                            continue
```

I am thinking why don't we check in line 124 if the leave_type is not active we are just gonna skip it, as I think it won't matter anyways to have a value for `allocation_remaining_display` too since we are not using this allocation anymore. What do you think?